crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -69,19 +69,9 @@ fn apply_normal_mapping(
     world_normal: vec3<f32>,
     double_sided: bool,
     is_front: bool,
-#ifdef VERTEX_TANGENTS
```

Comment:
just calling out for other reviewers - this function is no longer called unless VERTEX_UVS / VERTEX_TANGENTS are active so the behaviour changes in the non-tangent case don't matter.

but also @pcwalton - array_texture.wgsl needs updating for this api change.

Reply:
Good catch, fixed.